_test-server/app.js

```diff
@@ -12,9 +13,9 @@ const app = module.exports = express({
 	hasHeadCss: true,
 	layoutsDir: path.join(process.cwd(), '/layout'),
 	viewsDirectory: '/_test-server/views',
+	partialsDirectory: process.cwd(),
```

Comment:
could you experiment with removing all the hacky stuff in `make serve` that copies a load of templates. This may fix that